bump.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Appreciate the idea of adding a script, but did you know you can actually do this with npm? It does a git commit by default too, I prefer without, here's the command:
```
npm --no-git-tag-version version patch
```